angular/projects/spark-angular/src/lib/components/sprk-stack/sprk-stack.stories.ts

```diff
@@ -0,0 +1,421 @@
+  template: `
+    <sprk-stack splitAt="tiny">
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--three-tenths@xs"
+      ></div>
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--seven-tenths@xs"
+      ></div>
+    </sprk-stack>
+  `,
+});
+
+stackSplitLayoutThreeTenths.story = {
+  name: 'Stack/Split - 30/70',
+};
+
+export const stackSplitLayoutMixed = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-stack splitAt="tiny">
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--fourth@xs sprk-u-AbsoluteCenter"
+      >
+        <p class="sprk-b-TypeBodyOne">
+          fourth
+        </p>
+      </div>
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--half@xs sprk-u-AbsoluteCenter"
+      >
+        <p class="sprk-b-TypeBodyOne">
+          half
+        </p>
+      </div>
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--fourth@xs sprk-u-AbsoluteCenter"
+      >
+        <p class="sprk-b-TypeBodyOne">
+          fourth
+        </p>
+      </div>
+    </sprk-stack>
+
+    <sprk-stack splitAt="tiny">
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--sixth@xs sprk-u-AbsoluteCenter"
+      >
+        <p class="sprk-b-TypeBodyOne">
+          sixth
+        </p>
+      </div>
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--sixth@xs sprk-u-AbsoluteCenter"
+      >
+        <p class="sprk-b-TypeBodyOne">
+          sixth
+        </p>
+      </div>
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--sixth@xs sprk-u-AbsoluteCenter"
+      >
+        <p class="sprk-b-TypeBodyOne">
+          sixth
+        </p>
+      </div>
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--flex@xs sprk-u-AbsoluteCenter"
+      >
+        <p class="sprk-b-TypeBodyOne">
+          flex
+        </p>
+      </div>
+    </sprk-stack>
+
+    <sprk-stack splitAt="tiny">
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--two-fifths@xs sprk-u-AbsoluteCenter"
+      >
+        <p class="sprk-b-TypeBodyOne">
+          two-fifths
+        </p>
+      </div>
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--fifth@xs sprk-u-AbsoluteCenter"
+      >
+        <p class="sprk-b-TypeBodyOne">
+          fifth
+        </p>
+      </div>
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--fifth@xs sprk-u-AbsoluteCenter"
+      >
+        <p class="sprk-b-TypeBodyOne">
+          fifth
+        </p>
+      </div>
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--fifth@xs sprk-u-AbsoluteCenter"
+      >
+        <p class="sprk-b-TypeBodyOne">
+          fifth
+        </p>
+      </div>
+    </sprk-stack>
+
+    <sprk-stack splitAt="tiny">
+      <div
+        sprkStackItem
+        class="sprk-o-Stack__item--half@xs"
+      >
+        <sprk-stack
+          splitAt="tiny"
+          itemSpacing="medium"
```

Comment:
the  itemSpacing="medium" needs to be removed, i realized it's not supposed to be there in mine too